src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/OnlineEnvironment.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.VisualStudio.OnlineEnvironment.Client
+{
+    /// <summary>
+    /// Specifies various constant values needed in the online environment.
+    /// </summary>
+    internal static class OnlineEnvironment
+    {
+        /// <summary>
+        /// Name of the Solution Explorer view responsible for showing the contents of a
+        /// solution (as opposed to the contents of a folder).
+        /// </summary>
+        public const string LiveShareSolutionView = "LiveShareSolutionView";
+
+        /// <summary>
+        /// Guid indicating that the selected node in Solution Explorer is a project.
+        /// </summary>
+        public static readonly Guid SolutionViewProjectGuid = new Guid("F9806588-A88E-4429-8BFD-228795DB3894");
```

Comment:
Is this a GUID that we created or we have it from Visual Studio ?

Reply:
This is something that I copied from the Visual Studio code. It's always something that's defined on their end and components that want to extend depend on it.

Reply:
Hopefully they do not change it. If they do we will catch in E2E testing right?

Reply:
Yeah, eventually we'll catch it in our E2E test as well. 
I'm working on test plans and infra on the side. 

These things are not supposed to be changed, specifically we are already doing things like this in our Current Visual Studio implementation. 

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct#L261